core/enrollments/apps.py

```
@@ -4,3 +4,12 @@
 class EnrollmentsConfig(AppConfig):
     default_auto_field = 'django.db.models.BigAutoField'
     name = 'enrollments'
+    label = 'core_enrollments'
+    verbose_name = 'Enrollments'
+
+    def ready(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `ready`

Add return type annotation: `None`

(ANN201)